Add force-reactivate checkbox to order

pc-coholic opened the pull request:
The form used is not exactly meant to be used here - but it works and has exactly the same functionality... So in order to avoid code duplication: here it is :-)

raphaelm commented:
This sounds like a small recipe for later problems, for example there's the long-standing idea to add a "date of payment" to the ConfirmPaymentForm. Maybe rename it to an abstract name and create two empty subclasses, so the code structure makes clear where it is used?

pc-coholic commented:
- Renamed the `ConfirmPaymentForm` to `ForceQuotaConfirmationForm`
- Added `class ConfirmPaymentForm(ForceQuotaConfirmationForm)`
- Added `class ReactivateOrderForm(ForceQuotaConfirmationForm)`
